### ISSUE TITLE
p-tables use table-layout: auto when scrollable is false and so columns cannot be resized smaller than their content.

### ISSUE THREAD

**Issue by mconner** [primefaces/primeng#11393]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**

https://stackblitz.com/edit/github-cnsbpe-fzpkgp?file=src/app/app.component.html

**Current behavior**

When a p-table is defined as [scrollable]="false", you can't resize columns smaller than their content, even if you have autoLayout="false".  If you set it to [scrollable]="true", then it works as expected.

**Expected behavior**

Resizing of columns smaller than their content should be possible if it is not scrollable, just as it is when it is scrollable.

**Minimal reproduction of the problem with instructions**

See stackblitz:
1. Attempt to resize the columns. They cannot be resized smaller than the content.
2. Click Not Scrollable to switch to Scrollable.
3. Note that you can resize the column width to less than the content.
In short, scrollable == false results effectively in table-layout: auto.

**What is the motivation / use case for changing the behavior?**

The table should not need to be scrollable just to allow this.  In particular, I have a table with a paginator where the rows per page is updated based on row height and container height, so there is no need for it to scroll, but I have to allow for it to get full resizing, which results in a scrollbar appearing and disappearing for no good reason.

**Please tell us about your environment:**

Windows 10, IntelliJ, npm

* **Angular version:** 13.x

* **PrimeNG version:** 13.3.2

* **Browser:** [Edge]

 
* **Language:** [TypeScript ]

* **Node (for AoT issues):** `node --version` =   v16.14.0

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap? 
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,
